hangman.py

```diff
@@ -137,9 +204,18 @@ def match_with_gaps(my_word, other_word):
         _ , and my_word and other_word are of the same length;
         False otherwise: 
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    my_word_new = my_word.replace(' ', '')
```

Comment:
But I can see that you already done that replacement on 1 level up. Why do you do it here as well?